src/style.css

```diff
@@ -1,3 +1,74 @@
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  width: 100%;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.nav {
+  margin: 0;
+  padding: 0;
+}
+
+.nav__list {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  align-items: center;
+}
+
+.nav__item {
+  margin-left: 20px;
+  text-transform: uppercase;
+  list-style: none;
+}
+
+.nav__item:nth-child(1) {
+  margin-left: 0;
+}
+
+a {
+  display: block;
+  align-items: center;
+  line-height: 60px;
+}
+
+.logo {
+  display: flex;
+}
+
+.nav__link {
+  font-family: Roboto, serif;
```

Comment:
The font-family declaration includes 'Roboto' and 'serif'. Ensure that 'Roboto' is correctly loaded from Google Fonts as specified in the HTML file. If 'Roboto' fails to load, the fallback will be a serif font, which might not match the intended design.